packages/react/src/__tests__/ReactElementClone-test.js

```diff
@@ -371,6 +385,11 @@ describe('ReactElementClone', () => {
     ) {
       expect(clone.ref).toBe(element.ref);
       expect(clone.props).toEqual({foo: 'ef'});
+    } else if (
+      gate(flags => flags.enableRefAsProp && !flags.disableStringRefs)
+    ) {
+      expect(clone.ref).toBe(element.ref);
```

Comment:
Can't access `element.ref` when `enableRefAsProp` is `true` without wrapping in `toWarnDev`

```js
expect(() => {
  expect(clone.ref).toBe(element.ref);
}).toWarnDev(' Accessing element.ref was removed in React 19');
```

Reply:
Fixed pushed